### ISSUE TITLE
add capability to delete users

### ISSUE THREAD

**Issue by davewhiteland** [buggyrace/buggy-race-server#128]
Can already by deactivating them — which _effectively_ deletes them from the interface — but ought to allow for admins to remove a user or users entered in error. It's a cascade delete: need to remove task texts and buggy too (which is why deactivating them was implemented first; not just because it's simpler, but it's less catastrophic if done in error/maliciously).

**Comment by davewhiteland** [comment]
In user's `models.py` add `cascade`:

```python
buggies = db.relationship("Buggy", backref="users", lazy=True, cascade="all, delete")
```

to allow the cascade delete to work: see 7a5c849 for races.

**Comment by davewhiteland** [comment]
implemented in d5fd560
